db/migrate/20201217084605_add_paused_from_to_pay_subscriptions.rb

```diff
@@ -0,0 +1,5 @@
+class AddPausedFromToPaySubscriptions < ActiveRecord::Migration[4.2]
+  def change
+    add_column :pay_subscriptions, :paused_from, :datetime
```

Comment:
Since Paddle is the only one that needs a `paused_from` column (at the moment), what do you think about just putting this in the json `data` column instead?

Reply:
If it makes merging this PR easier, it's fine for me to put this in the `data` column for now. It's always possible to do a refactoring later if an additional payment provider needs a pause functionality.